charts/cadc-tap/templates/tap-deployment.yaml

```diff
@@ -74,6 +107,13 @@ spec:
                   name: "cadc-tap"
                   key: "pgpassword"
             {{- end }}
+            {{- if .Values.cloudsql.enabled }}
+            - name: "uwspassword"
+              valueFrom:
+                secretKeyRef:
+                  name: "cadc-tap"
+                  key: "uwspassword"
```

Comment:
This would change to `uws-db-password` if my previous suggestion sounded good. The internal `$uwspassword` environment variable is fine and doesn't need to change, although I probably would pedantically change the environment variables used in this chart to use all-caps names like `TAP_UWS_DB_PASSWORD` and `TAP_DB_PASSWORD` (for `pgpassword`). But this is just me being perfectionist and you can ignore that. :)

Reply:
Sure all that makes sense, changes applied (uws-db-password and the all-caps env names)